f5router/f5router.go

```diff
@@ -501,6 +540,8 @@ func (r *F5Router) process() bool {
 			r.processRouteAdd(ru)
 		} else if ru.Op() == routeUpdate.Remove {
 			r.processRouteRemove(ru)
+		} else if ru.Op() == routeUpdate.Update {
```

Comment:
We should use different types here, bind and unbind, then processRouteUpdate can be split into multiple functions.

Reply:
Good idea I'll implement it.